.github/workflows/cd.yml

```diff
@@ -191,6 +191,14 @@ jobs:
           tag: "template-tag-list"
           allowUpdates: true
 
+      - name: sync up fx-core fallback templates
+        run: |
+          rm -r ./templates/fallback
+          node ./scripts/download-templates-zip.js $(git tag -l templates*)
```

Comment:
current vm branch may have such tag like templates@2.0.1-alpha.3498543894.0 
will this can work??

Reply:
Removed.